app/src/main/res/values/strings.xml

```diff
@@ -310,4 +310,8 @@
     <string name="saved_image">Saved image</string>
     <string name="saving_image">Saving image…</string>
     <string name="permission_denied">Permission denied</string>
+    <string name="inbox_activity_replies">Replies</string>
+    <string name="inbox_activity_mentions">Mentions</string>
+    <string name="inbox_activity_messages">Messages</string>
+    <string name="bookmarks_activity_saved">Saved</string>
```

Comment:
There is already two Saved string resources, maybe use an existing? Or is it not applicable in this context?

Reply:
Probably. It is applicable in this context but there already were duplicating string resources previously for different contexts so i thought it would be better to separate them to leave some flexibility.